src/components/ErrorsContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component, PropTypes } from 'react';
+import FieldConnect from './FieldConnect';
+import ErrorField from './ErrorField';
+
+class ErrorsContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.getErrors = this.getErrors.bind(this);
+    }
+    getErrors() {
+        const { getAllErrors } = this.context;
+        const errorsObj = getAllErrors();
+        let errors = [];
+        Object.keys(errorsObj).forEach((errorIndex) => {
+            errorsObj[errorIndex].forEach(error => errors.push(error));
```

Comment:
You should add support for object and array. If you use ObjectField or ListField you will get object or array of array etc.
#### Example
```js
{
    foo: [
        { 
            bar: ['some error'],
            foo: [
                bar: ['some error2']
            ]
        }
    ]
}
```

Reply:
Done